app/utils/vrf_key.go

```diff
@@ -225,23 +202,39 @@ func LoadVRFKey(keyFilePath string) (*VRFKey, error) {
 	return vrfKey, nil
 }
 
-func InitializeVRFKey(config *cfg.Config) (vrfPubKey sdkcrypto.PubKey, err error) {
-	pvKeyFile := config.PrivValidatorKeyFile()
-	if err := os.MkdirAll(filepath.Dir(pvKeyFile), 0o700); err != nil {
-		return nil, fmt.Errorf("could not create directory %q: %w", filepath.Dir(pvKeyFile), err)
-	}
+// LoadOrGenVRFKey initializes a VRF key and returns its public key.
+// If loadPath is specified, it loads the VRF key file at the specified
+// path. Otherwise, it generates a new VRF key, whose entropy is randomly
+// generated or obtained from the mnemonic, if provided.
+func LoadOrGenVRFKey(config *cfg.Config, loadPath string) (vrfPubKey sdkcrypto.PubKey, err error) {
+	var vrfKey *VRFKey
+	if loadPath != "" {
+		vrfKey, err = LoadVRFKey(loadPath)
```

Comment:
Is this standard for cosmos? I'd say it's a risk to store a private key as plain text in a file. So we should additionally have the option to do it as an ENV var.

Reply:
created an issue based on what we discussed: #368